universal-login-wallet/src/core/entities/WalletModalContext.ts

```diff
@@ -11,6 +10,6 @@ export interface TopUpModalProps {
 
 type ErrorMessageType = string;
 
-export type WalletModalPropType = Partial<ModalWaitingForProps> | TopUpModalProps | ErrorMessageType;
+export type WalletModalPropType = Partial<WaitingForProps> | TopUpModalProps | ErrorMessageType;
```

Comment:
Why Partial?

Reply:
Because not all values are set at the same moment.